src/native/external/wasi-sdk-p2.cmake

```diff
@@ -0,0 +1,45 @@
+# TODO: WASI-SDK doesn't provide a CMake toolchain file for `wasm32-wasip2`, so
+# we must provide it ourselves.  Once
+# https://github.com/WebAssembly/wasi-sdk/pull/430 is merged and included in an
+# upcoming release, we'll be able to remove this file and use that instead.
+
+# Cmake toolchain description file for the Makefile
+
+# This is arbitrary, AFAIK, for now.
+cmake_minimum_required(VERSION 3.4.0)
```

Comment:
make it 3.5.0, see also https://github.com/WebAssembly/wasi-sdk/pull/429#discussion_r1643165379